intro-HPC/gent/macros.tex

```diff
@@ -3,20 +3,22 @@
 \macro{\university}{UGent}
 \macro{\association}{AUGent}
 % HPC definitions
-\macro{\hpc}{HPC-UGent}
+\macro{\hpc}{UGent-HPC}
```

Comment:
That's your opinion, no consensus yet on #87...

But fine, we can easily roll this back.